mmdet/models/backbones/efficientnet.py

```diff
@@ -0,0 +1,369 @@
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmcv.cnn import (build_conv_layer, build_norm_layer, constant_init,
+                      kaiming_init)
+from mmcv.runner import load_checkpoint
+
+from mmdet.utils import get_root_logger
+from ..builder import BACKBONES
+from ..utils.activations import Swish
+from ..utils.se_block import SE
+
+
+def drop_path(x, drop_prob: float = 0., training: bool = False):
+    """Drop paths
+
+    Args:
+        x (int): Feature map input.
+        drop_prob (float): Drop path rate.
+            Default: 0.0
+        training (bool): Whether the layer is in train mode.
+            Default: False
+    """
+    if drop_prob == 0. or not training:
+        return x
+    with torch.no_grad():
+        keep_prob = 1 - drop_prob
+        shape = (x.shape[0],) + (1,) * (x.ndim - 1)
+        random_tensor = keep_prob + torch.rand(shape, dtype=x.dtype, device=x.device)
+        random_tensor = torch.floor(random_tensor)
+    output = x / keep_prob * random_tensor
+    return output
+
+
+class MBConv(nn.Module):
```

Comment:
We have an InvertedResidual class in mmdet/models/utils/inverted_residual.py. Try to use that.